Description: Ignore "[cups-deviced] PID 17962 (usb) crashed on signal 11!" errors in the log during the test suite. This seems to regularly happen on Debian/Ubuntu buildds.
Author: Martin Pitt <mpitt@debian.org>

Index: cups-1.6.0/test/run-stp-tests.sh
===================================================================
--- cups-1.6.0.orig/test/run-stp-tests.sh	2012-07-27 18:52:28.000000000 +0200
+++ cups-1.6.0/test/run-stp-tests.sh	2012-07-27 18:55:31.162105242 +0200
@@ -815,7 +815,7 @@
 fi
 
 # Error log messages
-count=`$GREP '^E ' /tmp/cups-$user/log/error_log | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' /tmp/cups-$user/log/error_log | grep -v '(usb) crashed on signal 11' | wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
 	$GREP '^E ' /tmp/cups-$user/log/error_log
